app/uk/gov/hmrc/tariffclassificationfrontend/views/create_liability.scala.html

```diff
@@ -0,0 +1,63 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.tariffclassificationfrontend.config.AppConfig
+@import uk.gov.hmrc.tariffclassificationfrontend.controllers.routes
+@import uk.gov.hmrc.tariffclassificationfrontend.models.Case
+@import uk.gov.hmrc.tariffclassificationfrontend.views.forms.components.RadioOption
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.forms.components.input_radiogroup
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.includes.main
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.partials.case_heading
+@import uk.gov.hmrc.tariffclassificationfrontend.models.LiabilityOrder
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.components.confirm_or_cancel_buttons
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.forms.components.input_text
+@import uk.gov.hmrc.tariffclassificationfrontend.models.LiabilityStatus
+@(form: Form[LiabilityOrder])(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+@main(bodyClasses = None, customTitle = Some(messages("page.title.create_liability"))) {
+
+    <h1 id="create_liability-heading" class="heading-xlarge">Liability information</h1>
+
+    @helper.form(action = routes.CreateLiabilityController.post(), 'id -> "create_liability-form") {
+        @helper.CSRF.formField
+
+        <h2 class="heading-large">What is the trader's name?</h2>
```

Comment:
I _think_ the accessible solution is to pass the title text into the input_text / input_radiogroup controls - rather than have the H2 tags

Reply:
It's weird, on the prototype it seems to have a title, label and a  hint for this field

Reply:
Fair enough.